lib/breadcrumbs_on_rails/railtie.rb

```diff
@@ -10,7 +10,7 @@ module BreadcrumbsOnRails
 
   class Railtie < Rails::Railtie
     ActiveSupport.on_load(:action_controller) do
-      include BreadcrumbsOnRails::ActionController
+      ::ActionController::Base.send :include, BreadcrumbsOnRails::ActionController
```

Comment:
This is nit picky but I think most would agree we should use parenthesis here: https://github.com/bbatsov/ruby-style-guide#method-invocation-parens